hack/generator/Makefile

```diff
@@ -19,7 +20,7 @@ manifests: $(CONTROLLER_GEN)
 
 .PHONY: gen-arm
 gen-arm: rm-apis build ; $(info $(M) running k8sinfra-gen gen…) @ ## Running gen
-	$(Q) ./bin/$(APP) gen azure-arm.yaml
+	$(Q) ./bin/$(APP) -v 1 gen azure-arm.yaml
```

Comment:
This is setting log verbosity right?

Is there a reason we want more than the default level of logging during the build?
Phrased another way: if there is information we think is useful to be displayed all the time would we rather just put that stuff in v=0 (which I think is the default?)

Reply:
Agreed - changing it back.